doc/whats_new/v0.20.rst

```diff
@@ -11,6 +11,11 @@ This release packs in a mountain of bug fixes, features and enhancements for
 the Scikit-learn library, and improvements to the documentation and examples.
 Thanks to our many contributors!
 
+.. warning::
+
+    Version 0.20 is the last version of scikit-learn to support Python 2.7 and Python 3.4.
+    Scikit-learn 0.21 will require Python 3.5 or higher.
```

Comment:
``higher`` -> ``newer`` to keep consistent? I'm fine with current version though.